src/main/resources/mcpi/api/python/modded/mcpi/event.py

```diff
@@ -42,4 +42,26 @@ def __repr__(self):
     @staticmethod
     def Post(entityId, message):
         return ChatEvent(ChatEvent.POST, entityId, message)
+    
+class ProjectileEvent:
+    """An Event related to projectiles (e.g. placed, removed, hit)"""
+    HIT = 0
+
+    def __init__(self, type, x, y, z, face, shooterName,victimName):
+        self.type = type
+        self.pos = Vec3(x, y, z)
+        self.face = face
+        self.shooterName = shooterName
```

Comment:
Im not sure shooter and victim is the right terminology.

originName, targetName?

Reply:
I agree with your proposal. Victim seems to be incorrect term when you hit a block :)